Source/WebCore/platform/Pasteboard.h

```diff
@@ -72,6 +72,8 @@ class FragmentedSharedBuffer;
 
 struct SimpleRange;
 
+static constexpr unsigned pasteboardExpireTime = 8*60;
```

Comment:
Nit -
```suggestion
static constexpr auto pasteboardExpirationDelay = 8_min;
```

And then `pasteboardExpirationDelay.seconds()` at the call site when calling `+dateWithTimeIntervalSinceNow:`.